include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -653,9 +655,9 @@ AffineConstraints<number>::make_consistent_in_parallel(
       constraints_to_make_consistent.add_indices(constrained_indices.begin(),
                                                  constrained_indices.end());
 
-      // 3) Clear and refill this constraint matrix.
+      // 3) Clear and refill this constraint matrix. Also resolve chains:
```

Comment:
Well that part technically is available first with #17623 :-)

Reply:
Fair enough :-) We'll get there!